sass/_theme_rst.sass

```diff
@@ -26,10 +26,30 @@
     margin-bottom: $base-line-height
     p.caption
       font-style: italic
+      font-family: inherit !important
+      font-size: inherit !important
+      color: $caption-color !important
+    
 
   div.figure.align-center
     text-align: center
 
+  caption
+    font-style: italic
+    // workarond a wyrm overide
+    font-family: inherit !important
+    font-size: inherit !important
+    color: $caption-color !important
```

Comment:
Same with `!important` here. We can target this selector with greater specificity to override wyrm rules.